addon/components/flexberry-map.js

```diff
@@ -77,6 +90,7 @@ export default Ember.Component.extend(
       this.set('_layer', map);
       this._addObservers();
       this._addEventListeners();
+      this.sendAction('leafletMapDidInit', map);
     },
 
     didInsertElement() {
```

Comment:
Create map's <div> inside init, and attach it to DOM later in didiInsertElement is really bad decision.
Whole component's wrapping <div> (this.$()) must be a leaflet map's container, and it should be initialized with leaflet strictly in didInsertElemet hook.

All init logic inside component itself & in nested chaild components (layers, controls) must wait (you should make a promise from map object passed to them, and force them to wait, or refactor their init logic some other way).